### DISCUSSION TITLE
Reply and Deleting of comments functionality in Discussion forum is not working as expected

### DISCUSSION THREAD

**Discussion by abhishekpnt** [Sunbird-Lern/Community#126] [Issues]
Hi Team,

While testing discussion forum in dev region we noticed that while adding comments/reply in a discussion, the first comment/reply will be added successfully. But while adding the second comment/reply, the api is getting failed with 400 BAD REQUEST error even though the payload is same for both api calls. 
<img width="1655" alt="image" src="https://github.com/Sunbird-Lern/Community/assets/116337484/a3d02d81-88d5-4212-9b47-25c1827d0bb6">
<img width="1217" alt="image" src="https://github.com/Sunbird-Lern/Community/assets/116337484/9c5f51ee-b42e-457e-a565-ee41728a06a3">

Also while deleting a comment , the API is getting failed with 504 GATEWAY TIMEOUT error. But if we manually refresh the page, the comment will be deleted
<img width="1680" alt="image" src="https://github.com/Sunbird-Lern/Community/assets/116337484/a8250b27-7839-40df-92ef-bd2fe64bba23">

Can you please help us on this?

CC:  @vinukumar-vs @rajeevsathish

**Comment by vinukumar-vs** [comment]
@abhishekpnt Can you share the logs of Discussion-MW. What is the reason for failure? 

Without for details of logs, it's difficult for anyone to answer.

[@rajeevsathish](https://github.com/rajeevsathish), [@Krishnaj20](https://github.com/Krishnaj20)

**Reply by Jayaprakash8887** [reply]
@abhishekpnt Kindly provide logs for further analysis or mark the ticket as answered if you were able to address the issue on your own. (With steps you did to address it).

**Reply by abhishekpnt** [reply]
@Jayaprakash8887 The issue still persists on dev environment. I am unable to find the logs for the above issue in dev graylog of discussion middleware

**Reply by Krishnaj20** [reply]
@AmiableAnil - Is this issue still open?

**Comment by vinukumar-vs** [comment]
@abhishekpnt Without logs, team can't help. I hope this issue is already resolved. So this can be closed now.
